Tests/Indicators/CorrelationInidicatorTests.cs

```diff
@@ -0,0 +1,209 @@
+            firstRenkoConsolidator.Dispose();
+            secondRenkoConsolidator.Dispose();
+        }
+
+        [Test]
+        public override void AcceptsVolumeRenkoBarsAsInput()
+        {
+            var indicator = CreateIndicator();
+            var firstVolumeRenkoConsolidator = new VolumeRenkoConsolidator(100000);
+            var secondVolumeRenkoConsolidator = new VolumeRenkoConsolidator(1000000);
+            firstVolumeRenkoConsolidator.DataConsolidated += (sender, renkoBar) =>
+            {
+                Assert.DoesNotThrow(() => indicator.Update(renkoBar));
+            };
+
+            secondVolumeRenkoConsolidator.DataConsolidated += (sender, renkoBar) =>
+            {
+                Assert.DoesNotThrow(() => indicator.Update(renkoBar));
+            };
+            int counter = 0;
+            foreach (var parts in GetCsvFileStream(TestFileName))
+            {
+                var tradebar = parts.GetTradeBar();
+                if (tradebar.Symbol.Value == "SPY")
+                {
+                    firstVolumeRenkoConsolidator.Update(tradebar);
+                }
+                else
+                {
+                    secondVolumeRenkoConsolidator.Update(tradebar);
+                    counter++;
+                }
+                if (counter >= 500)
+                    break;
+            }
+
+            Assert.IsTrue(indicator.IsReady);
+            Assert.AreNotEqual(0, indicator.Samples);
+            firstVolumeRenkoConsolidator.Dispose();
+            secondVolumeRenkoConsolidator.Dispose();
+        }
+
+
+
+
+        [Test]
+        public void EqualCorrelationValue()
+        {
+            var indicator = new CorrelationIndicator("testCorrelationIndicator", Symbols.AAPL, Symbols.SPX, 3);
+
+            for (int i = 0 ; i < 3 ; i++)
+            {
+                indicator.Update(new TradeBar() { Symbol = Symbols.AAPL, Low = 1, High = 2, Volume = 100, Close = i + 1 ,Time = _reference.AddDays(1 + i) });
+                indicator.Update(new TradeBar() { Symbol = Symbols.SPX, Low = 1, High = 2, Volume = 100, Close = i + 1, Time = _reference.AddDays(1 + i) });
+            }
+
+            Assert.AreEqual(1, (double)indicator.Current.Value);
+        }
+
+        [Test]
+        public void NotEqualCorrelationValue()
+        {
+            var indicator = new CorrelationIndicator("testCorrelationIndicator", Symbols.AAPL, Symbols.SPX, 3);
+
+            for (int i = 0; i < 3; i++)
+            {
+                indicator.Update(new TradeBar() { Symbol = Symbols.AAPL, Low = 1, High = 2, Volume = 100, Close = i + 1, Time = _reference.AddDays(1 + i) });
+                indicator.Update(new TradeBar() { Symbol = Symbols.SPX, Low = 1, High = 2, Volume = 100, Close = i + 2, Time = _reference.AddDays(1 + i) });
+            }
+            Console.WriteLine(indicator.Current.Value);
+            Assert.AreNotEqual(0, (double)indicator.Current.Value);
```

Comment:
- Use Lean's logger
- Use `AreEqual` with the expected value

Reply:
I left AreNotEqual as the test is Called: `NotEqualCorrelationValue`
The previous test is: `EqualCorrelationValue` and there is `AreEqual`.
Please let me know what to do in this situation, because in this case, we will get 2 the same tests with 2 different names.